src/functions/template-tags/general.php

```diff
@@ -1453,6 +1460,8 @@ function tribe_events_is_view_enabled( $view ) {
 	 * @return string|null Will return null on Bad Post Instances
 	 */
 	function tribe_events_get_the_excerpt( $post = null, $allowed_html = null, $skip_postdata_manipulation = false ) {
+		static $cache_excerpts = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 2157d11b0677256b21415cf8e076b796</sub>